src/chrome/content/rules/Mozilla.net.xml

```diff
@@ -24,40 +20,33 @@
 			- air
 			- code
 			- developer
-			- download		(→ ftp.mozilla.org)
+			- download
 			- download-installer
 			- fhr
 			- marketplace
 			- mozorg
 			- support
 			- versioncheck
 		- videos-cdn
-		- www			(→ www.mozilla.org)
+		- www
 
 	Mixed content:
 		- Images on hardhat.cdn.mozilla.net from www.mozilla.org *
 		- Ad on activations.cdn.mozilla.net from getpocket.com *
 	* Secured by us
 -->
-<ruleset name="Mozilla.net (partial)">
+<ruleset name="Mozilla.net">
 
 	<target host="*.mozilla.net" />
 		<test url="http://cdn.mozilla.net/browserquest/img/1/spritesheet.png" />
 		<test url="http://code.cdn.mozilla.net/fonts/fira.css" />
 		<test url="http://developer.cdn.mozilla.net/" />
+		<test url="http://download.cdn.mozilla.net/" />
+		<test url="http://www.mozilla.net/" />
 
```

Comment:
`<securecookie host=".+" name=".+" />`

Reply:
`securecookie` can be critical on wildacrds, but I think in this cases it's okay. Mozilla is migrating all services to preloaded HSTS and AFAIK the CDNs doesn't use any relevant cookies.